cffdump/crashdec.c

```diff
@@ -520,6 +521,286 @@ dump_cp_ucode_dbg(uint32_t *dbg)
 	}
 }
 
+static void
+dump_meq_reg_write(unsigned reg, uint32_t data, bool pipe)
+{
+	if (pipe) {
+		struct rnndecaddrinfo *info = rnn_reginfo(rnn_pipe, reg);
+		printf("\t\twrite %s (%02x) pipe\n", info->name, reg);
+		switch (reg) {
+		/* registers that ignore their payload */
+		case 0x81:
+		case 0x82:
+		case 0x84:
```

Comment:
maybe useful to add something in the xml instead of hard-coding here?  At least the offsets would be different on other generations.  And just updating xml as we discover new things seems useful?

Reply:
Do you mean adding a new attribute to the `regN` tags? Is it possible to pass that through easily? Or just matching on the decoded name here?

Thankfully we probably won't discover anything new, at least for a6xx, since we only care about things produced by the firmware and we can more or less grep for all the pipe regs it writes to.

Reply:
in case you missed it on irc, I pushed a few fixups to:

https://github.com/freedreno/envytools/commits/crashdec-decode-moar

The easy way to handle the "ignore payloads" was to introduce a special type that crashdec can check for.

Reply:
I added those fixups, and I also added a commit to sync the registers with mesa before adding more to make merging the changes less painful.